libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/FrameworkMediaDrm.java

```diff
@@ -327,6 +329,24 @@ public void restoreKeys(byte[] sessionId, byte[] keySetId) {
     mediaDrm.restoreKeys(sessionId, keySetId);
   }
 
+  @Override
+  @RequiresApi(Build.VERSION_CODES.Q)
+  public void removeOfflineLicense(byte[] keySetId) {
+    if (Util.SDK_INT < Build.VERSION_CODES.Q) {
+      throw new UnsupportedOperationException();
+    }
+    mediaDrm.removeOfflineLicense(keySetId);
+  }
+
+  @Override
+  @RequiresApi(Build.VERSION_CODES.Q)
+  public List<byte[]> getOfflineLicenseKeySetIds() {
+    if (Util.SDK_INT < Build.VERSION_CODES.Q) {
+      return ImmutableList.of();
```

Comment:
Thinking if we should also throw `UnsupportedOperationException()` exception incase of unsupported API level to match the pattern

Reply:
Yes you're right, that is easier to reason about and more consistent - done.

Reply:
Got it. Thank you!